server/models/reporthistory.js

```diff
@@ -0,0 +1,26 @@
+export default (sequelize, DataTypes) => {
+  const ReportHistory = sequelize.define('ReportHistory', {
+    reportBody: DataTypes.STRING,
+    authorId: DataTypes.INTEGER,
+    reporterId: DataTypes.INTEGER,
+    // typeId: DataTypes.INTEGER
```

Comment:
Why this commented code?

Reply:
Shouldn't be commented, I will fix it now.